README.md

```diff
@@ -550,7 +550,20 @@ server side.
 If writing your own gRPC clients in the language of choice consult
 gRPC guide on [cancellation](https://grpc.io/docs/guides/cancellation/#cancelling-an-rpc-call-on-the-client-side).
 
+### GRPC Error Status Codes
 
+Starting from release 24.09, the Triton Python gRPC client introduces support for gRPC error codes in streaming mode, enhancing error reporting capabilities. When this feature is enabled, the Triton server will return standard gRPC error codes and subsequently close the stream after delivering an error
```

Comment:
This feature is not specific to python client right? 
All streaming clients (C++ or user-generated gRPC clients) can add header ("triton_grpc_error": "true") to observe the similar behavior.

Reply:
Yes, should the docs change then be in server?
I assumed GRPC client would be a good place to put it as there was a lot of existing GRPC docs here

Reply:
The client location is good as all our client libraries live in this repo. We just need to update the language to say that this feature is available for all "clients"(c++  and python).

See how we document the[ compression support](https://github.com/triton-inference-server/client?tab=readme-ov-file#compression-1)...  
We can do something similar.